.env.version

```
@@ -1,5 +1,6 @@
 # VERSION: https://semver.org/
 
 VERSION=5.13.0
+BUNDLE_VERSION=1
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The BUNDLE_VERSION key should go before the VERSION key

(UnorderedKey)